[PR-12263][SEO] Meta description for docs pages (#190)
talkable/talkable-docs: source/web_hooks/referral.rst

.. _web_hooks/referral:
.. include:: /partials/common.rst

.. meta::

>    :description: The Talkable Referral Webhook notifies your endpoint that your brand advocate’s referral status “Approved” or “Unblocked”.

Reviewer: ```suggestion
   :description: Referral Webhook notifies your endpoint that your brand Advocate’s referral status has become “Approved” or “Unblocked”.
```
